api/models/articles.model.js

```diff
@@ -69,3 +69,19 @@ exports.addComment = (comment, articleId) => {
       .then((data) => data.rows[0])
 }
 
+exports.deleteCommentByID = (commentId) => {
+  if (!commentId || isNaN(Number(commentId))) {
```

Comment:
would PSQL throw an error if the comment ID isn't valid?

Reply:
it would, I'll make the change.